### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hoffin**
**What this PR does / why we need it**:
Adds required @ symbol to az parameters value.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/867)

### CODE REVIEW

**Comment by JackQuincy** [comment]
what version of the az cli are you using? 

with version 2.0.7 I believe they moved to not requiring the @ symbol, it actual fails with it.  A better change would be to catalog the difference based on the az cli version.

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ $ az group deployment create \
     --name "<DEPLOYMENT NAME>" \
     --resource-group "<RESOURCE_GROUP_NAME>" \
     --template-file "./_output/<INSTANCE>/azuredeploy.json" \
-    --parameters "./_output/<INSTANCE>/azuredeploy.parameters.json"
+    --parameters "@./_output/<INSTANCE>/azuredeploy.parameters.json"
```
Comment (rjtsdl) [inline README.md]: This was removed intentionally. 
az cli 2.10 + don't need `@` anymore.

**Comment by rjtsdl** [comment]
@hoffin, please close this PR. Thx

File: README.md (hunk 1 above)
Comment (hoffin) [inline README.md]: I was using 2.0.9 and it was failing without. strange. closing.

**Pull request closed without merging**